.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -857,21 +856,18 @@ jobs:
         id: verify
         shell: bash
         run: |
-          if [ "${{ github.ref_type }}" = "tag" ]; then
-            AGENT_VERSION="$(cat VERSION | tr -d '\n')"
-            TAG="${{ github.ref_name }}"
-            DOCKER_VERSION_TAG="${TAG#v}"
-            if [ "${{ github.ref_name }}" = "v${AGENT_VERSION}" ]; then
-              echo "publish=true" >> "${GITHUB_OUTPUT}"
-              echo "tags=${DOCKER_VERSION_TAG},latest" >> "${GITHUB_OUTPUT}"
-            fi
-          else
-              echo "publish=true" >> "${GITHUB_OUTPUT}"
+          AGENT_VERSION="$(cat VERSION | tr -d '\n')"
+
+          if [ "${{ github.ref_type }}" = "tag" ] && [ "${{ github.ref_name }}" != "v${AGENT_VERSION}" ]; then
+            echo "tags=empty" >> "${GITHUB_OUTPUT}"
+          elif ${{ github.ref_name }} == "master"; then
               echo "tags=${{ github.sha }}" >> "${GITHUB_OUTPUT}"
+          else
```

Comment:
I think that it would be better to use something like that for tags for images which were created as part of PR builds: `pr-<num>-<commit sha>`. 

For example: `pr-1184-bc0610268a25ae80a4f8e8088814571fa4a30401`.

In case this results in too many tags, we could omit the commit sha (last part) from the tag.

And I imagine we may eventually also want up some clean up process / workflow which deletes PR tags which are older than X (e.g. 90) days.

Reply:
Regarding the naming - good point, I like that and will change it.

Ad repository cleaning - https://sentinelone.atlassian.net/browse/DTIN-3073

Reply:
> I think that it would be better to use something like that for tags for images which were created as part of PR builds: `pr-<num>-<commit sha>`.
> 
> For example: `pr-1184-bc0610268a25ae80a4f8e8088814571fa4a30401`.
> 
> In case this results in too many tags, we could omit the commit sha (last part) from the tag.
> 
> And I imagine we may eventually also want up some clean up process / workflow which deletes PR tags which are older than X (e.g. 90) days.

Changes commited.